modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/SignatureType.scala

```diff
@@ -0,0 +1,11 @@
+/** Rosetta Build Once. Integrate Your Blockchain Everywhere.
+  *
+  * The version of the OpenAPI document: 1.4.12 Contact: team@openapitools.org
+  *
+  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech). https://openapi-generator.tech
+  */
+
+package org.tessellation.rosetta.server.model
+
+case class SignatureType(
```

Comment:
I mean, this is duplicated. Is it intended?

Reply:
I get what you are trying to say now. Changed to `SignatureTypeValue`.